src/__fixtures__/default/schema.json

```diff
@@ -1,1941 +1,31 @@
 {
-  "description": "Schema for Atlassian Document Format.",
```

Comment:
This file is changed because it doesn't work properly with ajv 8, at least not in strict mode. It also yields different errors.

It is a simplified version more suitable for test-cases.